src/Raven.Server/Documents/Patch/ScriptRunner.cs

```diff
@@ -175,7 +174,7 @@ public SingleRun(DocumentDatabase database, RavenConfiguration configuration, Sc
                 ScriptEngine.SetValue("Raven_Min", new ClrFunctionInstance(ScriptEngine, Raven_Min));
                 ScriptEngine.SetValue("Raven_Max", new ClrFunctionInstance(ScriptEngine, Raven_Max));
 
-                ScriptEngine.SetValue("convertJsTimeToTimeSpanString", new ClrFunctionInstance(ScriptEngine, ConvertJsTimeToTimeSpanString));
```

Comment:
What happened when a 4.1 client talks to us?

Reply:
then it will fail.. i thought it was fine to drop it in 4.2 but you're right, we need to keep it